### PR TITLE
feat: use different colors on text selection/highlight dark/light mode

### PR DESCRIPTION
**Pull request by yaroslavhoretskyi**
Light Mode (old):
image
(new)
image

Dark Mode:
image

### CODE REVIEW

**Review by MandyMeindersma — CHANGES_REQUESTED** [review]
I would love to see the opacity change, like so: 
image

**Comment by yaroslavhoretskyi** [comment]
I love it, I'll add it!

**Comment by MandyMeindersma** [comment]
Oh sorry @yaroslavhoretskyi I wasn't talking about the colour. 

I was talking about the opacity. Like how the "G"s aren't cut off, you know what I mean?

image

image

My guess is that you would have to use a rgb(#,#,#,0.5) method to get it to work but maybe ask around tonight to see if someone else has a better idea.

**Comment by dgmouris** [comment]
Sorry I updated this branch while I was reading it!

File: assets/style.css [hunk 1]
```diff
@@ -13,3 +13,16 @@ p + p {
 .nuxt-scroll-indicator {
   @apply bg-gradient-to-r to-secondary from-blue-700;
 }
+
+/* We use this color just one time */
+.light {
```
Comment (arashsheyda) [inline assets/style.css:18]: I think it's easier to use tailwind syntax here, for example:
```css
::selection: {
	@apply bg-[#d4b5d4] dark:bg-primary
}
```

**Comment by MandyMeindersma** [comment]
@yaroslavhoretskyi I wanted to check in to see if you still wanted to give tis a shot :)

**Comment by yaroslavhoretskyi** [comment]
@MandyMeindersma, thanks for the ping, I just got lost in all the messages in my email.. I'd like to finish.

**Comment by MandyMeindersma** [comment]
Absolutely! No rush :)

File: assets/style.css (hunk 1 above)
Comment (yaroslavhoretskyi) [reply assets/style.css:18]: Thanks for your comment. I just realized that if we want to have different colors for different modes, we should use something like this.
```
::selection {
  @apply bg-secondary; // for light mode
}

.dark ::selection {
  @apply bg-primary;; // for dark mode
}
```
or if we want to use only one color for each mode, we can use your solution.
Let me know what you think about it.

Comment (arashsheyda) [reply assets/style.css:18]: @yaroslavhoretskyi Ah yes you're right, thanks

**Comment by MandyMeindersma** [comment]
Weird @yaroslavhoretskyi the package files are still showing for me. 

It almost looks like the bot is kind of readding stuff since there is a change on that file. I wonder if there is a way to remove all your changes from the file?

Or honestly copy and paste your changes to another branch ahhaha

**Comment by MandyMeindersma** [comment]
#349 

Sorry 😢

**Review by MandyMeindersma — APPROVED** [review]

**Comment by MandyMeindersma** [comment]
Thank you so much @yaroslavhoretskyi !!!

Sorry there were bot issues! I have turned off the bot so that won't happen again!

Thank you so much for making the website better!

**Comment by MandyMeindersma** [comment]
@allcontributors add @yaroslavhoretskyi for code

**Pull request merged**